### ISSUE TITLE
Post today's matches at the start of each day

### ISSUE THREAD

**Issue by pmonks** [pmonks/futbot#1]
### Description of Problem:
When it is a new day, I want the list of available soccer matches occurring that day (across all available competitions) to be posted to a Discord channel, so I can plan my viewing schedule accordingly.

### Potential Solutions:
Timer-triggered logic that pulls matches from the [football-data.org API `matches` resource](www.football-data.org/documentation/quickstart#available-resources) (potentially using a [filter](www.football-data.org/documentation/quickstart#filtering) to restrict to today's matches only, though that appears to be the default behaviour of the `matches` endpoint), formats that data into a pretty chat post, then sends that post to the (configurable) Discord server & channel.

### Potential Complications:
* [Discord's message formatting capabilities](https://support.discord.com/hc/en-us/articles/210298617-Markdown-Text-101-Chat-Formatting-Bold-Italic-Underline-) are (being as polite as possible...) "rudimentary", so this message will be a file attachment in a format that supports rich formatting options (specifically, tables) (see decision #7 for the full explanation of this).  HTML will be the initial format of this file attachment, but if that proves to be unacceptable we'll try PDF next.

**Comment by pmonks** [comment]
Discord on iOS is unable to preview HTML file attachments itself, and iOS doesn't provide any easy way of viewing an HTML file that gets downloaded (i.e. to the phones "Files" app).

tl;dr - HTML is a non-starter, and PDF is a better option (Discord *is* able to natively render PDF files itself, as soon as one is tapped).

**Comment by pmonks** [comment]
PDF generation added in r588cf524a4edfd4e355352dfd8a6362fc8c19466.

**Comment by pmonks** [comment]
I believe this is functional as of rev d84580272dbe1db44c02e985578bfa0dd45e036f, but awaiting tomorrow midnight UTC for a system testing to confirm.

**Comment by pmonks** [comment]
This was tested and confirmed to work today, so closing.
